src/utils/textHighlight/TextHighlight.vue

```diff
@@ -49,9 +49,19 @@ export default {
     },
     computed: {
         highlightedText() {
-            let regex = new RegExp(`(?<!<[^>]*)(${this.highlight})`, 'gi');
+            const highlight = this.highlight;
 
-            return this.text.replace(regex, `<span class="${this.highlightClass}">${this.highlight}</span>`);
+            const normalCharsRegex = /^[^A-Za-z0-9]+$/g;
+
+            const newHighLight = normalCharsRegex.test(highlight)
+                ? `\\${highlight}`
+                : highlight;
+
+            const highlightRegex = new RegExp(`(?<!<[^>]*)(${newHighLight})`, 'gi');
```

Comment:
It looks like the negative lookbehind feature [is not supported by Safari until 16.4 is released](https://caniuse.com/js-regexp-lookbehind). Testing this on Safari 16.3, the highlight feature completely breaks. Is there any way we can accomplish this without using lookbehind? Otherwise, we may have to wait until after 16.4 to take this.